dev/RadioMenuFlyoutItem/RadioMenuFlyoutItem.idl

```diff
@@ -14,6 +14,12 @@ unsealed runtimeclass RadioMenuFlyoutItem : Windows.UI.Xaml.Controls.MenuFlyoutI
 
     static Windows.UI.Xaml.DependencyProperty IsCheckedProperty{ get; };
     static Windows.UI.Xaml.DependencyProperty GroupNameProperty{ get; };
+
+    [MUX_DEFAULT_VALUE("false")]
```

Comment:
New APIs need to be added in [MUX_PUBLIC_V2] block

Reply:
ok, done

Reply:
Not quite -- this makes the attribute apply only to the next member. You need to put it in { } around the thing:

```
        [MUX_PUBLIC_V2]
        {
            [MUX_DEFAULT_VALUE("winrt::ControlsResourcesVersion::Version1")]
            ControlsResourcesVersion ControlsResourcesVersion{ get; set; };

            static Windows.UI.Xaml.DependencyProperty ControlsResourcesVersionProperty{ get; };
        }

```

Reply:
@jevansaks  I've updated this. We need to add the metadata test to catch these type of issues.